app/models/category.rb

```diff
@@ -6,4 +6,7 @@ def recent_videos
     self.videos.limit(6)
   end
 
+  def self.selection
+    Category.all.map { |category| [category.name, category.id] }
```

Comment:
Since this is view related logic, it doesn't go in the model. You should put that logic in a view helper method.